docs/usage.md

```diff
@@ -9,37 +9,6 @@ Using Action Scheduler requires:
 1. scheduling an action
 1. attaching a callback to that action
```

Comment:
I assume this 3rd ToC item (`attaching a callback to that action`) can be removed, as it's covered in "Scheduling an Action" section.

Reply:
@slaFFik Thanks for the suggestion. PR updated.